packages/main/src/themes/ColorPaletteItem.css

```diff
@@ -43,13 +50,21 @@
 	border-radius: 0.1875rem;
 }
 
-.ui5-cp-item:hover:not(:focus) {
-	border: 1px solid var(--sapGroup_ContentBackground);
+.ui5-cp-item:hover:not(:focus),
+:host(:not([on-phone])) .ui5-cp-item.ui5-cp-selected {
 	border-radius: var(--_ui5_color-palette-item-hover-inner-border-radius);
 	box-sizing: border-box;
 }
 
-:host(:not([_disabled]):not([on-phone])) .ui5-cp-item:focus{
+:host([selected]:not([_disabled]):not([on-phone]):not(:hover):focus) .ui5-cp-item.ui5-cp-selected,
+:host(:not([selected]):not([_disabled]):not([on-phone]):hover:focus) .ui5-cp-item, 
+:host([selected]:not([_disabled]):not([on-phone]):hover:focus) .ui5-cp-item.ui5-cp-selected {
+	border: var(--_ui5_color-palette-item-selected-focused-border);
+}
+
+.ui5-cp-item.ui5-cp-selected:focus,
+:host(:not([_disabled]):not([on-phone])) .ui5-cp-item:focus,
```

Comment:
this two selectors are confusing? we are trying to target something when it has the attribute and when it doesnt. do we need these selectors at all?

Reply:
missed that while testing, im removing it, thanks